libminifi/test/unit/StagingQueueTests.cpp

```diff
@@ -92,7 +92,7 @@ TEST_CASE("Modify and overflow triggered automatic commit", "[TestStagingQueue4]
   }
   queue.modify([] (MockItem& item) {
     // a new item has been allocated
-    REQUIRE(item.data_ == "");
+    REQUIRE(item.data_.empty());
```

Comment:
catch deconstructs the expression and logs the left-and right-hand side of an equality on failure, whereas with the change we only learn that it is not empty

Reply:
True, this is one of the more questionable clang-tidy checks, but in this case, I think it makes sense:
* start out with an empty queue
* add something
* commit
* check that the (newly started) queue is empty.

So in this particular case, I would leave it as `.empty()`, but I'll change it if you think `== ""` would/could give us useful information.

Reply:
True, this is one of the more questionable clang-tidy checks, but in this case, I think it makes sense:
* start out with an empty queue
* add something
* commit (explicitly or implicitly)
* check that the (newly started) queue is empty.

So in this particular case, I would leave it as `.empty()`, but I'll change it if you think `== ""` would/could give us useful information.